cloud/deployment/inspect/inspect.go

```diff
@@ -92,39 +96,45 @@ const (
 	notApplicable = "N/A"
 )
 
-func Inspect(wsID, deploymentName, deploymentID, outputFormat string, client astro.Client, coreClient astrocore.CoreClient, out io.Writer, requestedField string, template bool) error {
+func Inspect(wsID, deploymentName, deploymentID, outputFormat string, platformCoreClient astroplatformcore.CoreClient, coreClient astrocore.CoreClient, out io.Writer, requestedField string, template bool) error {
 	var (
-		requestedDeployment                                                        astro.Deployment
+		requestedDeployment                                                        astroplatformcore.Deployment
 		err                                                                        error
 		infoToPrint                                                                []byte
 		deploymentInfoMap, deploymentConfigMap, additionalMap, printableDeployment map[string]interface{}
 	)
 	// get or select the deployment
-	requestedDeployment, err = deployment.GetDeployment(wsID, deploymentID, deploymentName, true, client, coreClient)
+	requestedDeployment, err = deployment.GetDeployment(wsID, deploymentID, deploymentName, true, platformCoreClient, coreClient)
 	if err != nil {
 		return err
 	}
 
-	if requestedDeployment.ID == "" {
+	if requestedDeployment.Id == "" {
 		fmt.Printf("No Deployments found in workspace %s\n", ansi.Bold(wsID))
 		return nil
 	}
-
-	// get core deployment
-	coreDeployment, err := deployment.CoreGetDeployment(wsID, "", requestedDeployment.ID, coreClient)
+	// create a map for deployment.information
+	deploymentInfoMap, err = getDeploymentInfo(requestedDeployment)
 	if err != nil {
 		return err
 	}
-
-	// create a map for deployment.information
-	deploymentInfoMap, err = getDeploymentInfo(&requestedDeployment, coreDeployment)
+	// create a map for deployment.configuration
+	deploymentConfigMap, err = getDeploymentConfig(&requestedDeployment, platformCoreClient)
 	if err != nil {
 		return err
 	}
-	// create a map for deployment.configuration
-	deploymentConfigMap = getDeploymentConfig(&requestedDeployment)
 	// create a map for deployment.alert_emails, deployment.worker_queues and deployment.astronomer_variables
```

Comment:
is this change correct?
```suggestion
	// create a map for deployment.alert_emails, deployment.worker_queues and deployment.environment_variables
```

Reply:
yeah should be environment_variables